src/components/CardPickerCard.js

```diff
@@ -18,6 +18,10 @@ type CardPickerCardProps = {
    * The icon is limited in size to 60x60 pixels.
    */
   defaultIconUri: string | null;
+   /**
+   * The description for scanner type in the card.
+   */
+  desc: string | null;
```

Comment:
Can we rename "desc" to "description"? Both here in the component and in the CardPickerItem class too.

In addition, the comment shouldn't mention scanner types because the SUIT component is a generic card that can be used for anything.

Reply:
Can we put following comment for the description field ?
-A descriptive summary to supplement the card contents

Reply:
I think we're going to eventually make the description visible at all times instead of being a tooltip... maybe just say "An optional description of the card's contents"?